sdk/keyvault/Azure.Security.KeyVault.Administration/readme.md

```diff
@@ -0,0 +1,51 @@
+# Azure KeyVault Administration client library for .NET
+
+Content forthcoming
+
+## Getting started
+
+Content forthcoming
+
+### Prerequisites
+
+Content forthcoming
+
+### Install the package
+
+Content forthcoming
+
+### Authenticate the client
+
+Content forthcoming
+
+## Key concepts
+
+Content forthcoming
+
+## Examples
+
+Content forthcoming
+
+## Troubleshooting
+
+Content forthcoming
```

Comment:
Just as an FYI: this section and "Next Steps" are pretty boilerplate by design. See the existing Key Vault sections for examples.